docs_src/index.html.md

```diff
@@ -0,0 +1,469 @@
+and easy setup and management of continuous, automated deployment of a variety
+of project types on any virtual private server. The project is used, built, and
+maintained with ❤️ by [UBC Launch Pad](https://www.ubclaunchpad.com), UBC's
+student-run software engineering club.
+
+This site primarily documents how to set up and use Inertia - to learn more
+about the project, check out our [GitHub repository](https://github.com/ubclaunchpad/inertia)!
+
+# Getting Started
+
+## Installation
+
+> MacOS users can install the CLI using [Homebrew](https://brew.sh):
+
+```shell
+brew install ubclaunchpad/tap/inertia
+```
+
+> Windows users can install the CLI using [Scoop](http://scoop.sh):
+
+```shell
+scoop bucket add ubclaunchpad https://github.com/ubclaunchpad/scoop-bucket
+scoop install inertia
+```
+
+> To build and install the CLI from source:
+
+```shell
+go get -u github.com/ubclaunchpad/inertia
+```
+
+The Inertia command line interface, or "CLI", can be installed from a few package
+managers such as Homebrew and Scoop. For other platforms, you can 
+[download the appropriate binary from the Releases page](https://github.com/ubclaunchpad/inertia/releases).
+
+You can also build Inertia from source, though this requires Golang to be
+installed.
+
+To verify your installation, try running `inertia --help` - this should output
+some helpful text about Inertia.
+
+Run `inertia --version` to verify that you have the
+[latest release](https://github.com/ubclaunchpad/inertia/releases/latest).
+
+## Setup
+
+> This will generate a configuration file inside your repository:
+
+```
+cd /path/to/project
+inertia init
+```
+
+To set up Inertia, you must first navigate to your project directory, which
+must be a git repository. If Inertia cannot detect your project type, it will
+prompt for more information.
+
+<aside class="warning">
+<b>Do not commit the generated configuration file</b> - add it to your
+<code>.gitignore</code>!
+</aside>
+
+## Project Configuration
+
+> An example `inertia.toml`:
+
+```toml
+version = "test"
+project-name = "my_project"
+build-type = "dockerfile"
+build-file-path = "dockerfiles/Dockerfile.web"
+
+# ... other stuff
+```
+
+> To change a setting, you can edit the configuration file directly, or run:
+
+```shell
+inertia config set ${parameter} ${new_value}
+```
+
+Your Inertia configuration, stored in `inertia.toml` by default, contains a few
+key pieces of information.
+
+Parameter         | Description
+----------------- | -----------
+`version`         | This should match the version of your Inertia CLI, which you can see by running `inertia --version`. It is used to determine which version of the [Inertia daemon](https://cloud.docker.com/u/ubclaunchpad/repository/docker/ubclaunchpad/inertia/) to use.
+`project-name`    | The name of the project you are deploying.
+`build-type`      | This should be either `dockerfile` or `docker-compose`, depending on what you are using.
```

Comment:
I would say "which" instead of "what".

Reply:
(To explain - to me, "which" implies that you are either using Dockerfile or Docker Compose, whereas "what" implies there might be something external that determines which you pick.)

Reply:
yep! that makes sense, at least for now (since we dont support anything else)